DecipherNow/gm-fabric-dashboard
# Refactor: Notification SCSS

Resolves #1046 

## Proposed Changes

  - There isn't really any way to actually refactor this SCSS into styled-components, as we need to target react-notification-system's classnames in order to style them. Instead, I broke out the SCSS into chunks and put them in their own style folder as sort of faux-styled-components. Until injectGlobal is factored out of the styleVariables file, I had to add another injectGlobal in `Notification.js`.
  
## How to Test

  - Don't change the index.html meta tag and make sure the notifications are rendered and styled properly.
